### PR TITLE
osquery 1.5.3

### PR DESCRIPTION
**Pull request by theopolis**
Update osquery to 1.5.3!

### CODE REVIEW

File: Library/Formula/osquery.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Osquery < Formula
   homepage "https://osquery.io"
   # pull from git tag to get submodules
   url "https://github.com/facebook/osquery.git",
-      :tag => "1.5.2",
-      :revision => "225a14660abeb1071fff58e73cc753d54037c6ae"
+      :tag => "1.5.3",
+      :revision => "2a71162b0cfb040a3606538857f512ea8894b9e7"
   revision 1
```
Comment (DomT4) [inline Library/Formula/osquery.rb]: Can kill this line :wink:

Comment (theopolis) [inline Library/Formula/osquery.rb]: Line ded

**Comment by tdsmith** [comment]
Travis error looks real:

```
[  8%] Generating ../generated/tables_additional/xprotect_reports.cpp
/usr/bin/python /tmp/osquery20150929-20993-9l7ofz/tools/codegen/gentable.py /tmp/osquery20150929-20993-9l7ofz/specs/darwin/xprotect_reports.table /tmp/osquery20150929-20993-9l7ofz/generated/tables_additional/xprotect_reports.cpp 
Traceback (most recent call last):
  File "/tmp/osquery20150929-20993-9l7ofz/tools/codegen/gentable.py", line 24, in <module>
    from utils import platform
  File "/private/tmp/osquery20150929-20993-9l7ofz/tools/codegen/../tests/utils.py", line 18, in <module>
    import psutil
ImportError: No module named psutil
make[2]: *** [generated/tables_additional/xprotect_reports.cpp] Error 1
make[1]: *** [osquery/CMakeFiles/osquery_additional_amalgamation.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by ablyler** [comment]
@Homebrew/owners looks like this job might be hung up on brew bot: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/version=mavericks/33893/console it has been sitting at `brew update-test` for over an hour.

**Comment by DomT4** [comment]
Sigh. Give me a sec.

**Comment by DomT4** [comment]
Fixed.

**Comment by ablyler** [comment]
@DomT4 thanks!

**Comment by DomT4** [comment]
And fixed this failure:

```
==> Downloading https://homebrew.bintray.com/bottles/thrift-0.9.2.el_capitan.bottle.1.tar.gz
==> Retrying download
==> Downloading https://homebrew.bintray.com/bottles/thrift-0.9.2.el_capitan.bottle.1.tar.gz
Error: Failed to download resource "thrift"
Download failed: https://homebrew.bintray.com/bottles/thrift-0.9.2.el_capitan.bottle.1.tar.gz
```

Everything else passed. Future builds of this PR should go green now.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/157787706d2d4749a352ee6a48706194553ca0d2. Thank you for your contribution to Homebrew @theopolis!

**Pull request closed without merging**